decompiler/extractor/main.cpp

```diff
@@ -219,7 +219,20 @@ ExtractorErrorCode compile(const fs::path& iso_data_path, const std::string& dat
 }
 
 void launch_game() {
-  system(fmt::format("\"{}\"", (file_util::get_jak_project_dir() / "../gk").string()).c_str());
+  const std::string filepaths[2] = {
+      fmt::format("{}", (file_util::get_jak_project_dir() / "out\\build\\Release\\bin\\gk.exe").string()),
+      fmt::format("{}", (file_util::get_jak_project_dir() / "build/game/gk").string())};
```

Comment:
In fact I think this will break releases since neither of these folders are ever going to be present.

Reply:
I agree that it would be better to add a new CLI flag instead. However, I'm a little confused on why the filepath in launch_game() was originally hardcoded?